css-three-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  width: 1400px;
+  height: 600px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+}
+
+.container-2 {
+  width: 1000px;
+  margin: 0 auto;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  width: 100%;
+  height: 100px;
+  color: white;
+  font-family: Lato, sans-serif;
+  font-weight: normal;
+}
+
+.row {
+  display: flex;
+  width: 100%;
```

Comment:
Not needed

Reply:
Do you mean just the width is not needed?
If I take out the whole thing then it becomes a column of the images and text instead of a row w